doc/classes/Viewport.xml

```diff
@@ -220,6 +220,7 @@
 			If [code]true[/code], the viewport will not receive input events.
 		</member>
 		<member name="gui_embed_subwindows" type="bool" setter="set_embed_subwindows_hint" getter="get_embed_subwindows_hint" default="false">
+			If [code]true[/code], the viewport wiil embed windows.
```

Comment:
Describe what embedding subwindows does more in detail:

```suggestion
			If [code]true[/code], the viewport wiil embed [Window]s. This means [Window]s will [i]not[/i] be actual windows from the window manager's point of view, and they will not be able to leave the main window's viewport. If [code]false[/code], windows will be created separately, allowing them to be moved outside the main window (and alt-tabbing through them if desired).
```

Reply:
Added more detail and explained the different behavior of root- and sub-viewports.